index.html

```diff
@@ -2940,7 +2940,7 @@ <h3>Lifecycle Details</h3>
 The [=verifier=] <a href="#lifecycle-details">validates</a> claims.
           </li>
           <li>
-The [=verifier=] applies valid claims</a>.
+The [=verifier=] applies valid claims.
```

Comment:
```suggestion
The [=verifier=] applies valid [=claims=].
```

Reply:
@TallTed I added a link as you suggested. Thank you :)